include/aspect/parameters.h

```diff
@@ -166,6 +166,7 @@ namespace aspect
     double                         start_time;
     double                         CFL_number;
     double                         maximum_time_step;
+    bool                           use_smoothing;
```

Comment:
Or maybe better just `use_artificial_viscosity_smoothing`, since it's presumably a technique that can be used for any other artificial viscosity scheme as well.